packages/eas-cli/src/devices/udids.ts

```diff
@@ -0,0 +1,9 @@
+export function isValidUDID(udid: string): boolean {
+  if (udid.length === 25) {
+    return /[A-Z0-9]{8}-[A-Z0-9]{16}/.test(udid);
+  } else if (udid.length === 40) {
+    return /[a-z0-9]{40}/.test(udid);
```

Comment:
It seems that UUIDs are always hexadecimal characters only, so maybe we can have a more strict test?
```suggestion
    return /[a-f0-9]{40}/.test(udid);
```